src/bytes_mut.rs

```diff
@@ -1540,6 +1540,43 @@ impl PartialEq<Bytes> for BytesMut {
     }
 }
 
+impl Into<Vec<u8>> for BytesMut {
```

Comment:
Ppl usually write `From` implementations because there is already a built-in implementation of `Into` for everything `T: From` -> https://doc.rust-lang.org/std/convert/trait.Into.html#impl-Into%3CU%3E

Reply:
Thanks!

I would fix this.

Reply:
Done!